lib/std/math/acosh.zig

```diff
@@ -29,15 +29,15 @@ fn acosh32(x: f32) f32 {
 
     // |x| < 2, invalid if x < 1 or nan
     if (i < 0x3F800000 + (1 << 23)) {
-        return math.log1p(x - 1 + math.sqrt((x - 1) * (x - 1) + 2 * (x - 1)));
+        return math.log1p(x - 1 + @sqrt((x - 1) * (x - 1) + 2 * (x - 1)));
     }
     // |x| < 0x1p12
     else if (i < 0x3F800000 + (12 << 23)) {
-        return math.ln(2 * x - 1 / (x + math.sqrt(x * x - 1)));
+        return @log(2 * x - 1 / (x + @sqrt(x * x - 1)));
```

Comment:
`math.ln` and `@log` are not equivalent. `ln` is base `e` and `log` is base `10`. You would need to divide by `@log(e)`.

Reply:
`@log` does refer to the natural logarithm. See https://ziglang.org/documentation/0.9.1/#log.

Reply:
Thanks, I should have checked before commenting. Sorry for that!

Reply:
It's still a fair point; we may want to play with the naming to make it less ambiguous. Although we are a bit limited by libc here since we do need to match the symbol names, and libc has `log` as natural log.